File: Java/Miscellaneous/Review.java
package LTD_API;

import java.util.Date;

public class Review
{
    private String consumerName;
    private int stars;
    private Date createdAt;
    private String content;
    
    public String getConsumerName()
    {
        return consumerName;
    }
    public void setConsumerName(String consumerName)
    {
        this.consumerName = consumerName;
    }

    public int getStars()
    {
        return stars;
    }
    public void setStars(int stars)
    {
        this.stars = stars;
    }

    public Date getCreatedAt()
    {
        return createdAt;
    }
    public void setCreatedAt(Date createdAt)
    {
        this.createdAt = createdAt;
    }

    public String getContent()
    {
        return content;
    }
    public void setContent(String content)
    {
        this.content = content;
    }
}

File: PHP/Miscellaneous/Review.php
<?php
namespace LTD_API;

class Review
{
    private $consumerName;
    private $stars;
    //date format
    private $createdAt;
    private $content;
    
    public function getConsumerName()
    {
        return $this->consumerName;
    }
    public function setConsumerName($consumerName)
    {
        $this->consumerName = $consumerName;
    }

    public function getStars()
    {
        return $this->stars;
    }
    public function setStars($stars)
    {
        $this->stars = $stars;
    }

    public function getCreatedAt()
    {
        return $this->createdAt;
    }
    public function setCreatedAt($createdAt)
    {
        $this->createdAt = $createdAt;
    }

    public function getContent()
    {
        return $this->content;
    }
    public function setContent($content)
    {
        $this->content = $content;
    }
}

